### ISSUE TITLE
Pull mkDub.nix from this repository instead of bundling it?

### ISSUE THREAD

**Issue by vpanteleev-sym** [lionello/dub2nix#22]
Hi, is there any reason why `--output` copies `mkDub.nix`, instead of simply making the generated `.nix` file fetch it from this repository?

I think it would reduce clutter, but I'm not sure if there is some bigger disadvantage that I'm not seeing.

**Comment by lionello** [comment]
Yeah, that's certainly possible, but I thought it worthwhile to not depend on a 3rd party service when building a package. It's worth considering doing, however.
